tools/src/output.rs

```diff
@@ -375,6 +375,33 @@ pub fn generate_panel(
             F::S(
                 r#"<label class="panel-accel"><input type="checkbox" id="panel-accel-enable" checked="checked">Enable keyboard shortcuts</label>"#,
             ),
+            F::S(r#"<div><label class="panel-theme">"#),
```

Comment:
I assume we're just going to pick a theme instead of offering this selector :)

Reply:
Yeah, as much fun as changing things is, this takes up too much space, etc.